test/SlowTests/Issues/RavenDB_14464.cs

```diff
@@ -0,0 +1,148 @@
+            {
+                RunInMemory = false
+            });
+
+            using (var store = GetDocumentStore(new Options
+            {
+                ModifyDocumentStore = documentStore =>
+                {
+                    documentStore.Conventions.OperationStatusFetchMode = OperationStatusFetchMode.ChangesApi;
+                    documentStore.OnFailedRequest += (sender, args) => { };
+                },
+                Server = server,
+                RunInMemory = false
+            }))
+            {
+                new Company_ByName().Execute(store);
+                put_1500_companies(store);
+
+                WaitForIndexing(store);
+
+                var task = Task.Run(async () =>
+                {
+                    var operation = await store.Operations.SendAsync(new DeleteByQueryOperation(new IndexQuery { Query = $"FROM INDEX '{new Company_ByName().IndexName}'" }, new QueryOperationOptions
+                    {
+                        MaxOpsPerSecond = 1
+                    }));
+                    var serverToggled = false;
+
+                    while (store.WasDisposed == false)
+                    {
+                        try
+                        {
+                            if (serverToggled)
+                            {
+                                await operation.WaitForCompletionAsync();
+                            }
+                            else
+                            {
+                                await operation.WaitForCompletionAsync(TimeSpan.FromSeconds(1));
+                            }
+                            break;
+                        }
+                        catch (Exception e)
+                        {
+                            if (serverToggled == false)
+                            {
+                                Assert.True(e.GetType() == typeof(TimeoutException), "e.GetType() == typeof(TimeoutException)");
+                                Assert.True(e.Message.Contains("did not get a reply for operation"), "e.Message.Contains('did not get a reply for operation')");
+                                while (server.Disposed == false)
+                                {
+                                    await Task.Delay(500);
+                                }
+                                serverToggled = true;
+                            }
+
+                            Assert.True(e.GetType() == typeof(InvalidOperationException), "e.GetType() == typeof(InvalidOperationException)");
+                            Assert.True(e.Message.StartsWith("Could not fetch state of operation"), "e.Message.StartsWith('Could not fetch state of operation')");
+                            await Task.Delay(1000);
+                        }
+                    }
+                });
+
+                await Task.Delay(1000);
+                server = await ToggleServer(server);
+                await Task.Delay(5000);
```

Comment:
If there is no other choice than doing that, then tests like that should be moved to Stress.

Reply:
done